features/support/paths.rb

```diff
@@ -27,7 +27,7 @@ def path_to(page_name, *args) # rubocop:disable Metrics/AbcSize, Metrics/Cycloma
       provider_admin_authentication_providers_path
 
     when /^the sign up page for the "([^"]*)" plan$/
-      plan = Plan.find_by_name!($1)
+      plan = Plan.find_by!(name: $1)
```

Comment:
Why this? what's the difference?

Reply:
I think the original has been deprecated for a long time.

Reply:
An example warning from IDE:
![Screenshot from 2024-08-07 13-04-15](https://github.com/user-attachments/assets/30acbe2a-58a3-42ec-9658-7cfa6ec5bdc8)

Reply:
Nice, thanks.